src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -68,6 +68,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Containers.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Dapr.Tests" />
```

Comment:
We need to break this. What is this needed for?

Reply:
`await app.ExecuteBeforeStartHooksAsync(default);` in `DaprTests`

Reply:
Can we refactor the tests to not need to use an internal API?

Reply:
Using UnsafeAccessor instead